assets/js/util/index.js

```diff
@@ -135,10 +135,14 @@ export const readableLargeNumber = ( number, currencyCode = false ) =>  {
 	if ( false !== currencyCode && '' !== readableNumber ) {
 		const formatedParts = new Intl.NumberFormat( navigator.language, { style: 'currency', currency: currencyCode } ).formatToParts( number );
 
-		const decimal = formatedParts.find( part => 'decimal' === part.type ).value;
+		let decimal = formatedParts.find( part => 'decimal' === part.type );
+		if ( ! isUndefined( decimal ) ) {
```

Comment:
Two things to make the code more error-proof and simplify it:
1. To be safe, this should be: `! isUndefined( decimal ) && ! isUndefined( decimal.value )`
2. The other clause that you changed below can be combined with that since `decimal` isn't used anywhere else, and so there's no point to re-check whether it's not undefined. In other words, this should become: `! isUndefined( decimal ) && ! isUndefined( decimal.value ) && 1000 > number`, and then you can remove the `decimal = decimal.value` assignment and replace it with line 146 (assignment of `readableNumber`).